File: Python/2129. Capitalize the Title.py
class Solution:
    def capitalizeTitle(self, title: str) -> str:
        words = title.lower().split()
        print(words)
        for i, word in enumerate(words):
            if len(word) > 2:
                words[i] = word[0].upper() + word[1:]
        capitalized = " ".join(words)
        return capitalized
        

File: Golang/2129. Capitalize the Title.go
func capitalizeTitle(title string) string {
	words := strings.Split(strings.ToLower(title), " ")
	// fmt.Println(words)
	for i, word := range words {
		if len(word) > 2 {
			words[i] = strings.ToUpper(string(word[0])) + word[1:]
			// fmt.Printf("word[0]: %T, word: %T\n", word[0], word)
		}
	}
	capitalized := strings.Join(words, " ")
	return capitalized
}
